internal/metrics/metrics.go

```diff
@@ -12,7 +14,10 @@ const (
 	metricSubsystem = "loadtest"
 )
 
-const IterationMetricName = "form3_loadtest_iteration"
+const (
+	EnvMetricsEnrichment = "F1_METRICS_ENRICHMENTS"
```

Comment:
we have a package to encapsulate env based settings:
https://github.com/form3tech-oss/f1/tree/master/internal/envsettings